en/frames_vtol/vtol_tiltrotor_omp_hobby_zmo_fpv.md

```diff
@@ -0,0 +1,274 @@
+
+LTE antenna 1
+
+![LTE antenna 01](../../assets/airframes/vtol/omp_hobby_zmo_fpv/lte-antenna-01.jpg)
+
+LTE antenna 2 and RC receiver
+
+![LTE antenna 02](../../assets/airframes/vtol/omp_hobby_zmo_fpv/lte-antenna-02.jpg)
+
+## Software setup
+
+### Select airframe
+
+Open QGC and go under vehicle setup. Search for SYS_AUTO in the parameters tab. Set SYS_AUTOCONFIG to "Reset parameters to airframe defaults" and SYS_AUTOSTART to 13300. Reboot the vehicle.
+
+![SYS_AUTO](../../assets/airframes/vtol/omp_hobby_zmo_fpv/sys_auto.jpg)
+
+### Sensor calibration
+
+Do the sensor calibration. Make sure to select the correct orientation of the flight controller.
+
+### RC-Setup
+
+Calibrate you RC controller and setup the flight mode switch.
+
+### Actuator setup
+
+:::note
+Make sure the props are removed! The motors can start very easily in the actuators tab if a value gets changed in the wrong location.
+:::
+
+#### Tilt servos
+
+Switch the vehicle into manual mode. Either via the flight mode switch or type "commander mode manual" into the mavlink shell. Check if the motors are pointing upwards. If they are pointing forwards they need to be reversed. To do so select the checkbox under PWM-Main for each servo. Adjust the minimum or maximum value that the servo is pointing vertical up. Then type "commander transition" into the mavlink shell to adjust the horizontal position.
+
+![Tilt Servo adjustment](../../assets/airframes/vtol/omp_hobby_zmo_fpv/tilt-limits-01.jpg)
+
+#### Control surfaces
+
+Check with RC-Controller, if the actuators need to be reversed.
+- Roll stick to the right -> Right aileron up, left aileron down
+- Pitch stick to the back (fly upwards) -> both V-tail surfaces up
+- Yaw stick to the right -> both surfaces to the right
+
+Now adjust the trim value that all the surfaces are in neutral position.
+
+![Servo trim](../../assets/airframes/vtol/omp_hobby_zmo_fpv/servo-trim.jpg)
+
+#### Motor direction and orientation
+
+Make sure the props are removed!!!
+
+- Motor 1: Front left motor should spin CW
+- Motor 2: Front right motor should spin CCW
+- Motor 3: Rear motor should spin CCW
+
+If the motor spins in the wrong directions 2 of the 3 motor wires need to be swapped. The direction can not be changed in software since the ESCs are not using dshot.
+
+## First flight
+
+- Check CG (There are markings underneath the wing)
+- Check actuator orientations and neutral trim
+- Check tilt rotor reactions in stabilized
+    - Yaw the vehicle to the right (nose to the right) -> right motor should tilt down
+    - Yaw the vehicle to the left (nose to the left) -> left motor should tilt down
+- Check actuator reactions in stabilized (switch transition switch to enable fw mode)

```

Comment:
```suggestion
- Check actuator reactions in Stabilized (switch transition switch to enable FW mode)

```

Reply:
FYI, note that where possible, good to link to the mode you talk about - that's even better than using the proper name. I have modified the comment above and will merge it.